netann/node_announcement.go

```diff
@@ -1,14 +1,44 @@
 package netann
 
 import (
+	"encoding/hex"
+	"errors"
+	"image/color"
 	"net"
+	"regexp"
 	"time"
 
 	"github.com/lightningnetwork/lnd/keychain"
 	"github.com/lightningnetwork/lnd/lnwallet"
 	"github.com/lightningnetwork/lnd/lnwire"
 )
 
+var (
+	// validColorRegexp is a regexp that lets you check if a particular
+	// color string matches the standard hex color format #RRGGBB.
+	validColorRegexp = regexp.MustCompile("^#[A-Fa-f0-9]{6}$")
+)
+
+// ParseHexColor takes a hex string representation of a color in the
```

Comment:
This can now be used from the `lncfg` package.

Reply:
Good it got merged before this one!